test/to-port-to-wpts/misc2.js

```diff
@@ -416,19 +421,30 @@ describe("jsdom/miscellaneous", () => {
         </head>
         <body onload="loader()"></body>
       </html>`;
-    const doc = (new JSDOM(html, { runScripts: "dangerously" })).window.document;
-    const window = doc.defaultView;
+
+    const dom = new JSDOM(html, { runScripts: "dangerously" });
+    const { window } = dom;
+
+    let resolve;
+    let reject;
+    const result = new Promise((res, rej) => {
+      resolve = res;
+      reject = rej;
+    });
+
     // In JSDOM, listeners registered with addEventListener are called before
     // "traditional" listeners, so listening for "load" will fire before our
     // inline listener.  This means we have to check the value on the next
     // tick.
```

Comment:
Well, this just seems to be wrong. It looks like to me that per [spec](https://html.spec.whatwg.org/C/#event-handler-attributes), the `onload` content/traditional attribute should be called before ones added through JavaScript, if it is added first. Since the HTML one is added during parsing, it should be added pretty early I assume. Am I missing anything?

Reply:
Turns out, this has already been fixed.

The test was just not updated.